### DISCUSSION TITLE
gatsby-starter-wordpress-blog + WPGraphQL 1.6.7 - WPGatsby 2.0.2

### DISCUSSION THREAD

**Discussion by aggelosV1982** [gatsbyjs/gatsby#33950] [Help]
### Preliminary Checks

- [X] This issue is not a duplicate. Before opening a new issue, please search existing issues: https://github.com/gatsbyjs/gatsby/issues
- [X] This issue is not a question, feature request, RFC, or anything other than a bug report directly related to Gatsby. Please post those things in GitHub Discussions: https://github.com/gatsbyjs/gatsby/discussions

### Description

Hi, i´m trying gatsby-starter-wordpress-blog + WPGraphQL 1.6.7 - WPGatsby 2.0.2. Installed a new wordpress in local by fly wheel, and installed plugins WPGraphQL 1.6.7 - WPGatsby 2.0.2.
After this i installed gatsby-starter-wordpress-blog, edit gatsby-config.js url for http://localhostWP.local/graphql and use gatsby develop, but it show me this error:

ERROR #gatsby-source-wordpress_111002 

 gatsby-source-wordpress  Error: connect ECONNREFUSED 127.0.0.1:80   
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1161:16)

GraphQL request to http://localhostWP.local/graphql failed.

Please ensure the following statements are true
  - your WordPress URL is correct in gatsby-config.js
  - your server is responding to requests
  - WPGraphQL and WPGatsby are installed and active in your WordPress backend
  - Your WordPress debug.log does not contain critical errors

not finished createSchemaCustomization - 0.161s
not finished  gatsby-source-wordpress  ensuring plugin requirements are met - 0.147s

I have seen many videos about to start gatsby with wordpress headless, I did everything exactly like them in the videos but it show me the error.
Maybe it is the new version of the WPGraphQL plugin.  
if i try in localhostWP/graphql it show me that don´t have any query:

`{
"errors": [
{
"message": "GraphQL Request must include at least one of those two parameters: \"query\" or \"queryId\"",
"extensions": {
"category": "request"
}
}
],
"extensions": {
"debug": []
}
}`

### Reproduction Link

https://github.com/aggelosV1982/localhostWp

### Steps to Reproduce

1.install Wordpress in local by flywheel
2.install plugins WPGraphQL and WPGatsby from oficial plugins
3.Create new gatsby with starter-wordpress-blog.
4.edit gatsby-config.js, gatsby-source-wordpress, options>url and put http://localhostWP.local/graphql
5. run gatsby develop
...

### Expected Result

It must show me the wordpress post "hello world" and gatsby server running

### Actual Result

 gatsby-source-wordpress  Error: connect ECONNREFUSED 127.0.0.1:80   
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1161:16)

GraphQL request to http://localhostWP.local/graphql failed.

Please ensure the following statements are true
  - your WordPress URL is correct in gatsby-config.js
  - your server is responding to requests
  - WPGraphQL and WPGatsby are installed and active in your WordPress backend
  - Your WordPress debug.log does not contain critical errors

### Environment

```shell
System:
OS: Windows 10 64bits
CPU: Intel Core i5 8th gen
Binaries:
Node: 16.13.0
npm: 8.1.0
languages:
GatsbyJs, GraphQL, React Js, NodeJs,
```

**Comment by bzx** [comment]
I have the same issue. Ridiculous.

**Reply by aggelosV1982** [reply]
I resolved my issue, if are you using wsl2 and Windows 10, maybe is the same.
Local by flywheel is installed in Windows but the terminal is running in wsl2. Somebody told me that issue was closed port or wsl2 can't access it, i installed wordpress in my aws account and work it. I hope be helpfull for u

**Reply by iagobelo** [reply]
Instead of using local by flywheel, I used docker directly, and it worked. Thanks!
